templates/index.html

```diff
@@ -17,505 +19,653 @@
+        </div>
+        <div class="col-2 col-medium-1 col-small-1">
+          
+  <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_474,h_475/https://assets.ubuntu.com/v1/4c1992f8-lenovo-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_948,h_950/https://assets.ubuntu.com/v1/4c1992f8-lenovo-logo.png 2x" alt="" width="474" height="475" loading="auto|lazy">
+        </div>
+      </div>
     </div>
   </div>
-</div>
-
-<div class="p-strip--image is-dark is-slanted--top-right is-parallax" style="background-image:url('https://assets.ubuntu.com/v1/ce95be94-careers-guy.jpg');">
-  <div class="row u-vertically-center" style="height: 100vh;">
-    <div class="col-12 col-start-large-2 p-heading-highlight--left" data-animation="u-animation--slide-from-top">
-      <h2 class="p-heading-highlight--left__item p-heading--1">Local lifestyle. Global leadership.<br />Join Canonical to lead enterprise<br />open source.</h2>
-    </div>
+</section>
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule is-dark is-bold">
+    <h2><strong>Get to know Canonical</strong></h2>
   </div>
-</div>
-
-<div id="careers" class="p-strip--suru is-dark is-slanted--top-left js-active-target">
+</section>
+<section class="p-section">
   <div class="row">
-    <div class="col-4">
-      <div class="p-heading-highlight--bottom">
-        <h3 class="p-heading-highlight--bottom__item">Truly distributed</h3>
-      </div>
-      <p>Exceptional, self-motivated, organised and passionate people deserve the freedom to live where they want. Our teams travel regularly to meet colleagues and customers.</p>
-      <blockquote class="p-pull-quote--inverted">
-        <p class="p-pull-quote__quote">I have seen incredible places &ndash; an unexpected benefit of life at Canonical</p>
-      </blockquote>
-      <hr class="u-hide u-show--small"/>
-    </div>
-    <div class="col-4">
-      <div class="p-heading-highlight--bottom">
-        <h3 class="p-heading-highlight--bottom__item">Best in class</h3>
+    <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+      <hr class="p-rule is-dark is-bold">
+      <div class="row">
+        <div class="col-9">
+          <div class="p-block">
+            <h2 class="p-text--small-caps">Work with us</h2>
+          </div>
+          <p class="p-heading--2">Build the world’s best open source experience. <br class="u-hide--medium u-hide--small">Join our team.</p>
+          <p class="p-heading--2"><a href="https://canonical.com/careers/career-explorer">Browse careers&nbsp;›</a></p>
```

Comment:
```suggestion
          <p class="p-heading--2"><a href="/careers/career-explorer">Browse careers&nbsp;›</a></p>
```
You don't need the base url for internal links